end-to-end-test/tests/traders-hub/dashboard-real.spec.tsx

```diff
@@ -0,0 +1,40 @@
+import { test, expect } from '@playwright/test';
+import { TRADERS_HUB_URL, switchAccountType } from '../../utils';
+
+test.describe("Trader's Hub Dashboard", () => {
+    test.beforeEach(async ({ page }) => {
+        await page.goto(TRADERS_HUB_URL);
+        await switchAccountType(page, 'Demo', 'Real');
+    });
+
+    test('It should switch from Real to Demo', async ({ page }) => {
+        await switchAccountType(page, 'Real', 'Demo');
+        const demo_count = await page.getByText(/Demo/).count();
+        expect(demo_count).toBeGreaterThanOrEqual(1);
+    });
+
+    test('It should have the regulator switcher for low risk and no regulator switcher for high risk', async ({
+        page,
+    }) => {
+        const regulator_switcher = await page.getByText('Regulation:').count();
+        if (process.env.RISK_LEVEL === 'low_risk') {
+            expect(regulator_switcher).toEqual(1);
+        } else {
+            expect(regulator_switcher).toEqual(0);
+        }
+    });
+
+    test('It should switch to EU for low risk', async ({ page }) => {
+        if (process.env.RISK_LEVEL === 'low_risk') {
+            await page.getByText('EU', { exact: true }).first().click();
+            expect(await page.locator('text=Multipliers trading platform.')).toBeDefined();
```

Comment:
@jim-deriv nit: I suggest declaring a variable for the repeatedly used 'text=Multipliers trading platform.' string for better maintainability of this test file in future :)